conf/application.conf

```diff
@@ -18,7 +18,7 @@ appName="charities-registration-frontend"
 
 play.http.router = prod.Routes
 
-play.filters.headers.contentSecurityPolicy = "script-src 'self' 'unsafe-inline' localhost:9000 localhost:12345 localhost:8001 www.google-analytics.com http://www.googletagmanager.com https://www.googletagmanager.com https://tagmanager.google.com data:; style-src 'self' 'unsafe-inline' localhost:9000 localhost:8001 https://tagmanager.google.com https://fonts.googleapis.com; img-src 'self' 'unsafe-inline' localhost:9000 localhost:8001 https://ssl.gstatic.com www.gstatic.com https://www.google-analytics.com https://www.googletagmanager.com data:; font-src 'self' 'unsafe-inline' localhost:9000 localhost:8001 https://ssl.gstatic.com www.gstatic.com https://fonts.gstatic.com https://fonts.googleapis.com; frame-src 'self' https://www.googletagmanager.com;"
+play.filters.csp.CSPFilter = "script-src 'self' 'unsafe-inline' localhost:9000 localhost:12345 localhost:8001 www.google-analytics.com http://www.googletagmanager.com https://www.googletagmanager.com https://tagmanager.google.com data:; style-src 'self' 'unsafe-inline' localhost:9000 localhost:8001 https://tagmanager.google.com https://fonts.googleapis.com; img-src 'self' 'unsafe-inline' localhost:9000 localhost:8001 https://ssl.gstatic.com www.gstatic.com https://www.google-analytics.com https://www.googletagmanager.com data:; font-src 'self' 'unsafe-inline' localhost:9000 localhost:8001 https://ssl.gstatic.com www.gstatic.com https://fonts.gstatic.com https://fonts.googleapis.com; frame-src 'self' https://www.googletagmanager.com;"
```

Comment:
I am not 100% sure that we can just rename property like this , based on documentation https://www.playframework.com/documentation/2.8.x/CspFilter#Deprecation-of-SecurityHeaders.contentSecurityPolicy new property has a new structure in place, I am not sure how to test it but happy to try and merge and see if everything is working in one of the envs

Reply:
I've reverted after seeing new structure plus I think it needs an enabling line with this format